online-fishing-store/src/app/pages/profile-page/favorite-products/favorite-products-item/favorite-products-item.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="favorite-product">
+  <div class="favorite-product__wrapper">
+    <img
+      class="favorite-product__img"
+      [src]="favoriteProduct.img"
+      [alt]="productAltImgAttribute"
+    />
+    <span class="favorite-product__name">{{ favoriteProduct.name }}</span>      
+    <div class="favorite-product-price">
+      <span class="favorite-product-price">Цена:{{ favoriteProduct.price | currency: "USD":"symbol-narrow":"1.2" }}</span>
+    </div>
+  </div>
+  <div 
+      class="favorite-product__button"
+      (click)="removeFromFavorites(favoriteProduct)"
```

Comment:
For what you use argument here(favoriteProduct) if in this item you have only 1 product always